draft-ietf-tls-esni.md

```diff
@@ -757,6 +767,14 @@ If the server sends a NewSessionTicket message, the corresponding ECH PSK MUST
 be ignored by all other servers in the deployment when not negotiating ECH,
 including servers which do not implement this specification.
 
+## Backend Server Behavior {#backend-server-behavior}
+
+When the client-facing server accepts ECH, it forwards the ClientHelloInner to
+the backend server, who terminates the connection. The backend server confirms
+that ECH was accepted as follows. Per {{encrypted-client-hello}}, the
+ClientHelloInner contains an "ech" extension whose payload is a randomly
```

Comment:
Style nit: so far, the spec has been hyphenating "randomly-generated".

Reply:
Done.